platform-jobs/samza/post-publish-processor/src/main/java/org/ekstep/job/samza/service/PostPublishProcessor.java

```diff
@@ -59,12 +57,14 @@ public void initialize(Config config) throws Exception {
                 Arrays.asList(Platform.config.getString("post_publish_processor.contentTypes").split(",")) : Collections.emptyList();
         MAX_ITERATION_COUNT = (Platform.config.hasPath("max.iteration.count.samza.job")) ?
                 Platform.config.getInt("max.iteration.count.samza.job") : 1;
+        COURSE_TYPE = Platform.config.hasPath("post_publish_processor.courseTypes") ?
+                Arrays.asList(Platform.config.getString("post_publish_processor.courseTypes").split(",")) : Arrays.asList("CurriculumCourse");
```

Comment:
post_publish_processor.courseTypes is not added in property file

Reply:
config added.